tests/test_readFitsCatalog.py

```diff
@@ -88,8 +89,28 @@ def setUp(self):
         fitsTable = makeFitsTable()
         self.arr1 = Table(fitsTable[1].data).as_array()
         self.arr2 = Table(fitsTable[2].data).as_array()
+
         self.fitsTable = fitsTable
 
+    def _makeSchema(self, dtype):
```

Comment:
I think this could be static (doesn't need any instance variables) and should have at least a short descriptive docstring.

Reply:
I did some refactoring so that the same code is run in the code and tests.